website/pages/gamejam_en.md

```diff
@@ -3,7 +3,36 @@
 Get ready for our upcoming event, Global Game Jam!
 This year's edition of Global Game Jam will be held 29th - 31st January.
 Because COVID-19 is still wreaking havoc in the world, the jam will be held remotely in our [Discord server](https://discord.gg/eHgXYMS). Also check out our [Facebook event](https://fb.me/e/1SCmIAaRH).
-Join our site on Global Game Jams official [website](https://globalgamejam.org/2021/jam-sites/liu-game-jam)!
+
+**To participate**
+- Join our [Global Game Jam site](https://globalgamejam.org/2021/jam-sites/liu-game-jam).
+- Join our [Discord server](https://discord.gg/eHgXYMS).
+- Join our [Itch.io](https://itch.io/jam/global-game-jam-linkoping-2021)
```

Comment:
Kan vi skippa att ha det här som krav för "to participate" och bara ha discord? Känns fel att säga "nice, kul att du är intresserad. Börja med att skapa 3 olika konton"

Reply:
Det var Jespers förslag, så jag kan inte riktigt argumentera för eller emot. Det kan ju dock vara värt att åtminstone ha GGJ siten och Discord med då halva grejen med GGJ är att vara med på ett globalt jam och visa det genom GGJ's sida.

Reply:
Discord känns som ett måste, och ggj-sidan låter som ett krav för just det här jamet. Men krav på itch kan jag hålla med om att man kan skippa. Alla i en grupp behöver inte ett itchkonto för att ladda upp sina spel.

Reply:
Isf lägger jag till det som en "bra om du är med på men inte nödvändig"-punkt

Reply:
Vi kan pusha det mer på själva presentationen att vi vill ha folks spel på itch för att vi samlar på game jam-spel.